src/main.rb

```diff
@@ -0,0 +1,42 @@
+require_relative './app'
+
+data = { books: [], people: [], rentals: [] }
+
+puts 'Welcome to School Library your App!'
+
+def menu
+  puts 'Please choose an option by entering a number:'
+  puts '1 - List all books'
+  puts '2 - List all people'
+  puts '3 - Create a person'
+  puts '4 - Create a book'
+  puts '5 - Create a rental'
+  puts '6 - List all rentals for a given person id'
+  puts '7 - Exit'
+  gets.chomp
+end
+
+def main_menu(db_data) # rubocop:todo Metrics/CyclomaticComplexity
+  app = App.new
+  case menu
+  when '1'
+    app.list_all_books(db_data[:books])
+  when '2'
+    app.list_all_people(db_data[:people])
+  when '3'
+    app.create_person(db_data[:people])
+  when '4'
+    app.create_book(db_data[:books])
```

Comment:
- [ ] I commented your break to check if there's another error so after checking when I tried to create a new book it can't create a new book or a new person so kindly check the errors and try to solve the pronlems 💯💪🏻🚀

![image](https://user-images.githubusercontent.com/58553711/200719490-881e66c5-fa67-4642-ac53-2751d12b4a0d.png)

## It show that this private methods but it is not required to make it private so kindly remove the private from these methods to let your project works fine 💪🏻🔥🔥
![image](https://user-images.githubusercontent.com/58553711/200719224-4ff69fd9-b4bd-4725-b06a-7bcc6acf0fb5.png)